Resources/Private/Fusion/Prototypes/TrackingCode.GA.fusion

```diff
@@ -6,8 +6,10 @@ prototype(Neos.GoogleAnalytics:TrackingCode.GA) < prototype(Neos.GoogleAnalytics
         hasNoContainerId = ${String.isBlank(this.configuration.tagManager.id)}
     }
     
-    anonymizeString = ', {"anonymize_ip": true}'
-    anonymizeString.@if.anonymizeIp = ${this.configuration.analytics.anonymizeIp}
+    parameters = Neos.Fusion:DataStructure {
+        @process.json = ${Json.stringify(value)}
+        anonymize_ip = ${this.configuration.analytics.parameters.anonymizeIp}
+    }
```

Comment:
Almost there :)

When you do:

```
parameters = ${this.configuration.analytics.parameters}`
parameters.@process.json = ${Json.stringify(value)}
```

Then you can simply write `anonymize_ip: true` into the settings and it will be there.
That's what I meant when I said this way it's then completely free and easy to add new parameters.

Hope I don't discourage you with my proposals. But I think at the end the change will be really great :)

Reply:
Maybe add a hint on how to use it to the docs with the example of `anonymise_ip`

Reply:
Thank you. It's done